README.md

```diff
@@ -414,17 +456,34 @@ typedef NS_ENUM(BMACollectionViewLayoutMode, NSUInteger) {
 
 - `@synthesize` and `@dynamic` should each be declared on new lines in the implementation.
 
-- Attributes should be specific on what memory management should be used. Don't assume strong is default. Always us weak, strong, assign, or copy.
+- Do not be too verbose with attributes: no necessity to specify attributes used by default, but be specific when it's needed, and don't forget `atomic` is used by default
 
-- An atomic property should be marked as such, not left to the default value, which is atomic. This increases readability and awareness to other devs on the nature of this property.
+**Example:**
+```objc
+@interface BMAPerson
+@property (nonatomic) NSUInteger numberOfCommonPlaces;
+@property (nonatomic) NSArray *commonPlaces;
+@property (nonatomic, weak) id<BMAPersonDelegate> delegate;
+@end
+```
+**Not:**
+```objc
+@interface BMAPerson
+@property (nonatomic, assign) NSUInteger numberOfCommonPlaces; // assign is used by default for integers
+@property (strong) NSArray *commonPlaces; // shouldn't be atomic unless it's really needed, and strong is used by default for instance types
+@property (nonatomic) id<BMAPersonDelegate> delegate; // delegates are not owned
+@end
+```
 
-- Prefer atomic/nonatomic to be first in the attribute list. Keeps consistency around the codebase.
+- An atomic property should be marked as such, not left to the default value, which is `atomic`. This increases readability and awareness to other developers on the nature of this property.
 
-- Prefer using properties for all ivar access. There are many good reasons to do it, as stated [here for example](http://blog.bignerdranch.com/4005-should-i-use-a-property-or-an-instance-variable/). Direct ivar access should be justified. Refactor reckelessly legacy code which still uses instance variables.
+- Prefer `atomic`/`nonatomic` to be first in the attribute list: keep consistency around the codebase.
 
-- The only time when ivars should be used is dealloc and init methods. This is because in init and dealloc it's generally best practice to avoid side effects of setting properties directly, and because inside init, the object is still in a partial state.
+- Prefer using properties for all ivar access. There are many good reasons to do it, as stated [here for example](http://blog.bignerdranch.com/4005-should-i-use-a-property-or-an-instance-variable/). Direct ivar access should be justified. Refactor recklessly legacy code which still uses instance variables.
 
->For more information on using Accessor Methods in Initializer Methods and dealloc, see [here](https://developer.apple.com/library/mac/documentation/Cocoa/Conceptual/MemoryMgmt/Articles/mmPractical.html#//apple_ref/doc/uid/TP40004447-SW6).
+- The only time when ivars should be used is `dealloc` and `init` methods. This is because in `init` and `dealloc` it's generally best practice to avoid side effects of setting properties directly, and because inside `init`, the object is still in a partial state.
```

Comment:
Thanks, will be stated as following:
*The only time when ivars should be used is `dealloc` and `init` methods, and implementation of properties based on them*

``` objc
- (NSMutableDictionary *)cachedRequests {
    if (!_cachedRequests) {
        _cachedRequests = [NSMutableDictionary dictionary];
    }
    return _cachedRequests;
}
```